Utils/utils.ts

```diff
@@ -57,17 +57,29 @@ export class utils {
         }
     }
 
-    async click(selector: string | Locator, nextSelector: string, timeout?: number): Promise<void> {
+    async click(selector: string | Locator, nextSelector: string | Locator, timeout?: number): Promise<void> {
         try {
             let locator = typeof selector === 'string' ? await this.locator(selector) : selector;
             await locator.click();
-            await this.waitForSelector(nextSelector, { timeout: timeout })
+            
+            if (typeof nextSelector === 'string')
+            {
+                await this.waitForSelector(nextSelector, { timeout: timeout })
+            }
+            else if(!await nextSelector.isVisible())
+            {
+                throw Error('next locator is not visible after clicking the selector')
```

Comment:
we don't want to throw an error here if the selector is not visible, the idea is to wait for it within the interval timeout time sent

Reply:
it will wait by default to the selector to be visible, if not I want it to act is the page.waitForSelector dose to have consistency in the behavior of the selector not being visible on screen